service/ui.go

```diff
@@ -292,6 +292,15 @@ func (uis *UIServer) GetServiceApp() *gimlet.APIApp {
 		))
 	}
 
+	// Lobster
+	app.PrefixRoute("/lobster").Route("/").Handler(uis.lobsterPage).Get()
+	app.PrefixRoute("/lobster").Route("/{*}/").Handler(uis.lobsterPage).Get()
+	app.PrefixRoute("/lobster").Route("/{*}/{*}/").Handler(uis.lobsterPage).Get()
+	app.PrefixRoute("/lobster").Route("/{*}/{*}/{*}/").Handler(uis.lobsterPage).Get()
+	app.PrefixRoute("/lobster").Route("/{*}/{*}/{*}/{*}/").Handler(uis.lobsterPage).Get()
+	app.PrefixRoute("/lobster").Route("/{*}/{*}/{*}/{*}/{*}/").Handler(uis.lobsterPage).Get()
+	app.PrefixRoute("/lobster").Route("/{*}/{*}/{*}/{*}/{*}/{*}/").Handler(uis.lobsterPage).Get()
```

Comment:
I was unable to successfully catch all routes that fall under the lobster prefix in one statement. 
I attempted to access the mux struct directly to replace these lines but `mux` returns nil because of [this condition](https://github.com/evergreen-ci/gimlet/blob/main/app.go#L68) is not met. 

```
    if app != nil {
        mux, err := app.Router()
        if err != nil {
            grip.Error(err)
            return nil
        }
        if mux != nil {
            mux.HandleFunc(`/lobster/{rest:[a-zA-Z0-9=\-\/]+}`, uis.lobsterPage)
        }
    }
```

Reply:
This seems like an application of [AddPrefixRoute](https://github.com/evergreen-ci/gimlet/blob/2d28a05239b92c4dd562ab44ced731b2ac20cf5b/app_routing.go#L61)

I implemented it to use [here](https://github.com/evergreen-ci/evergreen/blob/8753b1aaa250d391995d26613ee3caf527495c66/service/ui.go#L341)

Reply:
thanks that did the trick